pkg/engine/update.go

```diff
@@ -323,6 +363,21 @@ func update(ctx *Context, info *planContext, opts planOptions, dryRun bool) (Res
 	return resourceChanges, res
 }
 
+func shortenPolicyPackPath(path string) string {
```

Comment:
This isn't specific to policy pack file paths, right? So perhaps we could rename this to "abbreviate file path" or "summarize path" or something more general like that? A doc comment would also be helpful explaining the specific conversion being performed, even though the method isn't exported.